2022-12-12 00:43:54,023 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 00:43:54,023 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 00:43:54,023 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 00:52:18,070 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 00:52:18,070 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 00:52:18,070 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:00:49,617 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:00:49,618 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:00:49,618 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:21:22,065 - DEBUG - ProfileManager.py - Line: 31 - Profile TestProfile01 already exist!
2022-12-12 01:23:40,065 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:23:40,065 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:23:40,066 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:27:49,635 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:27:49,636 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:27:49,636 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:28:30,899 - ERROR - ProfileManager.py - Line: 165 - ** Error: Exception caught at __main__!
2022-12-12 01:28:55,163 - ERROR - ProfileManager.py - Line: 166 - ** Error: Traceback (most recent call last):
  File "C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\Settings\ProfileManager.py", line 158, in _add_new_profile
    tree.write(f, pretty_print=True)
  File "src\lxml\etree.pyx", line 2072, in lxml.etree._ElementTree.write
  File "src\lxml\serializer.pxi", line 759, in lxml.etree._tofilelike
  File "src\lxml\etree.pyx", line 333, in lxml.etree._ExceptionContext._raise_if_stored
  File "src\lxml\serializer.pxi", line 682, in lxml.etree._FilelikeWriter.write
TypeError: write() argument must be str, not bytes

2022-12-12 01:40:26,452 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:40:26,453 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:40:26,453 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:45:30,198 - ERROR - ProfileManager.py - Line: 166 - ** Error: Exception caught at __main__!
2022-12-12 01:45:30,201 - ERROR - ProfileManager.py - Line: 167 - ** Error: Traceback (most recent call last):
  File "C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\Settings\ProfileManager.py", line 162, in _add_new_profile
  File "C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\Settings\ProfileManager.py", line 162, in _add_new_profile
  File "_pydevd_bundle\pydevd_cython_win32_39_64.pyx", line 1179, in _pydevd_bundle.pydevd_cython_win32_39_64.SafeCallWrapper.__call__
  File "_pydevd_bundle\pydevd_cython_win32_39_64.pyx", line 620, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.trace_dispatch
  File "_pydevd_bundle\pydevd_cython_win32_39_64.pyx", line 1095, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.trace_dispatch
  File "_pydevd_bundle\pydevd_cython_win32_39_64.pyx", line 1057, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.trace_dispatch
  File "_pydevd_bundle\pydevd_cython_win32_39_64.pyx", line 317, in _pydevd_bundle.pydevd_cython_win32_39_64.PyDBFrame.do_wait_suspend
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2021.2\plugins\python-ce\helpers\pydev\pydevd.py", line 1147, in do_wait_suspend
    self._do_wait_suspend(thread, frame, event, arg, suspend_type, from_this_thread)
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2021.2\plugins\python-ce\helpers\pydev\pydevd.py", line 1162, in _do_wait_suspend
    time.sleep(0.01)
KeyboardInterrupt

2022-12-12 01:45:30,202 - INFO - ProfileManager.py - Line: 176 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile01..xml
2022-12-12 01:45:30,204 - INFO - ProfileManager.py - Line: 176 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.TestProfile01..csv
2022-12-12 01:45:30,212 - INFO - ProfileManager.py - Line: 176 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.TestProfile01..csv
2022-12-12 01:45:30,218 - INFO - ProfileManager.py - Line: 176 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.TestProfile01..csv
2022-12-12 01:45:41,582 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:45:41,582 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:45:41,582 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:45:52,949 - WARNING - ProfileManager.py - Line: 32 - Profile TestProfile01 already exist. Msg:Profile name already exist!
2022-12-12 01:54:54,889 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:54:54,889 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:54:54,889 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:58:10,799 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:58:10,799 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:58:10,799 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:58:18,246 - WARNING - ProfileManager.py - Line: 32 - Profile TestProfile01 already exist. Msg:Profile name already exist!
2022-12-12 01:58:51,865 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 01:58:51,865 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 01:58:51,866 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 01:59:29,877 - INFO - ProfileManager.py - Line: 186 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile01.xml
2022-12-12 02:00:49,714 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 02:00:49,714 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 02:00:49,715 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 02:01:12,105 - INFO - ProfileManager.py - Line: 186 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile01.xml
2022-12-12 02:01:22,944 - INFO - ProfileManager.py - Line: 186 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.TestProfile01.csv
2022-12-12 02:01:32,438 - INFO - ProfileManager.py - Line: 186 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.TestProfile01.csv
2022-12-12 02:01:33,763 - INFO - ProfileManager.py - Line: 186 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.TestProfile01.csv
2022-12-12 02:22:27,810 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 02:22:27,811 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 02:22:27,811 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 02:22:31,280 - DEBUG - ProfileManager.py - Line: 38 - Creating backups for new profile
2022-12-12 02:22:33,890 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile01.xml
2022-12-12 02:22:34,374 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.TestProfile01.csv
2022-12-12 02:22:34,783 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.TestProfile01.csv
2022-12-12 02:22:35,549 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.TestProfile01.csv
2022-12-12 02:22:37,060 - DEBUG - ProfileManager.py - Line: 46 - Create new profile entry!
2022-12-12 02:22:43,635 - DEBUG - ProfileManager.py - Line: 38 - Creating backups for new profile
2022-12-12 02:22:43,635 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile02.xml
2022-12-12 02:22:43,637 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Actors.TestProfile02.csv
2022-12-12 02:22:43,639 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\SceneOrder.TestProfile02.csv
2022-12-12 02:22:43,641 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.csv -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\Db\Comments.TestProfile02.csv
2022-12-12 02:22:43,642 - DEBUG - ProfileManager.py - Line: 46 - Create new profile entry!
2022-12-12 02:23:10,405 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile02.xml
2022-12-12 02:23:10,407 - INFO - ProfileManager.py - Line: 205 - rename C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile02..xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml
2022-12-12 02:24:39,139 - INFO - Logger.py - Line: 47 - -----------------------------------
2022-12-12 02:24:39,139 - INFO - Logger.py - Line: 48 - Log system successfully initialised
2022-12-12 02:24:39,140 - INFO - Logger.py - Line: 49 - -----------------------------------
2022-12-12 02:25:33,683 - INFO - ProfileManager.py - Line: 197 - copy C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.xml -> C:\Users\Usuario\PycharmProjects\CKsQuestDialogManager\App\settings.TestProfile02.xml
